ros2doctor/ros2doctor/api/network.py

```diff
@@ -66,6 +66,11 @@ def check(self):
             return result
 
         has_loopback, has_non_loopback, has_multicast = _check_network_config_helper(ifcfg_ifaces)
+        if not has_loopback and not has_non_loopback:
```

Comment:
I would make this warning only happen on windows since it would be an actual error on other platforms if both of these were `False`. One way is to make this line check if the platform is windows. Another way would be to make `_check_network_config_helper()` return `None` for these values instead of `True` or `False` when there's no information, and then check if they're `None` here.